src/index.js

```diff
@@ -1,2 +1,54 @@
+/* eslint-disable linebreak-style */
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  // props가 없는 null 이면 {}로서 처리
+  Object.entries(props || {}).forEach(([key, value]) => {
+    // DOM의 속성은 소문자 처리
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    // 텍스트가 아니라 노드면
+    if (child instanceof Node) {
+      element.appendChild(child);
+    } else element.appendChild(document.createTextNode(child));
+  });
+
+  return element;
+}
+
+function render({ count = 0 } = {}) {
```

Comment:
```suggestion
function render({ count }) {
```

함수를 호출하는 쪽에서 초기값을 넣어주기 때문에 이정도로만 표현하셔도 괜찮을 것 같아요.